### ISSUE TITLE
Table translation fails if no trailing \cr

### ISSUE THREAD

**Issue by mcneney** [r-lib/pkgdown#664]
I'm trying to use pkgdown to build a website for the package [LDheatmap](https://github.com/mcneney/LDheatmap). build_reference() processes two of the help files successfully, then stops while processing the third with the following error:

`Reading 'man/GIMAP5.Rd'
Error in seq_len(last) : 
  argument must be coercible to non-negative integer`

(Not sure how to extract a smaller reproducible example to illustrate the problem.)

Any suggestions?

**Comment by batpigandme** [comment]
Can you please link to the problem file/section in the original R code as well as the Rd file?

Thanks

**Comment by mcneney** [comment]
The problem file is the Rd file
[https://github.com/mcneney/LDheatmap/blob/master/man/LDheatmap.Rd](https://github.com/mcneney/LDheatmap/blob/master/man/LDheatmap.Rd). I don't know what you mean by "section in the original R code". If you are asking which example in the Rd file is the problem, I don't know. I find it very hard to follow the traceback() or debug build_reference(). I've clipped in the traceback below.

`Reading 'man/GIMAP5.Rd'
Error in seq_len(last) : 
  argument must be coercible to non-negative integer
> traceback()
26: NextMethod()
25: structure(NextMethod(), class = class(x))
24: `[.tag`(contents, seq_len(last))
23: contents[seq_len(last)]
22: as_html.tag_tabular(.x[[i]], ...)
21: .f(.x[[i]], ...)
20: .Call(map_impl, environment(), ".x", ".f", "character")
19: purrr::map_chr(x, as_html, ...)
18: flatten_para(x, ...)
17: parse_section(x, "Format", ...)
16: as_data.tag_format(.x[[i]], ...)
15: .f(.x[[i]], ...)
14: .Call(map_impl, environment(), ".x", ".f", "list")
13: purrr::map(., as_data)
12: function_list[[i]](value)
11: freduce(value, `_function_list`)
10: `_fseq`(`_lhs`)
9: eval(quote(`_fseq`(`_lhs`)), env, env)
8: eval(quote(`_fseq`(`_lhs`)), env, env)
7: withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))
6: sections %>% purrr::map(as_data) %>% purrr::map(add_slug)
5: data_reference_topic(topic, pkg, examples = examples, run_dont_run = run_dont_run, 
       mathjax = mathjax)
4: .f(.x[[i]], ...)
3: .Call(map_impl, environment(), ".x", ".f", "list")
2: purrr::map(topics, build_reference_topic, pkg = pkg, lazy = lazy, 
       examples = examples, run_dont_run = run_dont_run, mathjax = mathjax)
1: build_reference()`

**Comment by mcneney** [comment]
A summer student working with me figured it out: I had several \tabular environments in my .Rd files where the last row did not end in a \cr (like I always do in LaTeX). When she puts \cr's at the end of all lines pkgdown runs fine. Is this behaviour expected?
Brad

**Comment by hadley** [comment]
It's clearly a bug.

**Comment by hadley** [comment]
@mcneney I don't see any reference to `\tabular` in https://github.com/mcneney/LDheatmap/blob/master/man/LDheatmap.Rd.
